buildpacks/python/python/validation.go

```diff
@@ -77,9 +77,14 @@ func (f *FunctionValidationLayer) Contribute(layer libcnb.Layer) (libcnb.Layer,
 			pythonPath = append(pythonPath, env)
 		}
 
-		cmd := exec.Command("python", "-m", "pyfunc", "check", "-s", f.applicationPath)
+		cmd := exec.Command(
+			"python",
+			"-m", "pyfunc", "check",
+			"-s", f.applicationPath,
+			"-m", f.module,
```

Comment:
@menehune23 Just double checking, the use of `"-m"` twice is correct/intentional?

Reply:
Yep, it's intentional. The first is for `python` to specify the `pyfunc` module. The latter is an argument for the `check` pyfunc subcommand.

Reply:
I'll re-organize the line endings to make this (hopefully) clearer